src/Redux/ConfigureStore.js

```diff
@@ -1,11 +1,14 @@
 import { createStore, combineReducers, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
 import logger from 'redux-logger';
 import booksReducer from './Books/Books';
 
```

Comment:
- [ ] After setting up the categories actions and reducer you are required to import the categoriesReducer here. Kindly do that.

Reply:
Done